UnityProject/Assets/Scripts/Objects/Machines/QuantumPad.cs

```diff
@@ -96,6 +101,8 @@ private void Start()
 			{
 				LavaLandManager.Instance.LavaLandBase2Connector = this;
 			}
+
+			//int randomNumber = Random.Range(1, 5);
```

Comment:
looks like you forgot to make it random, you can also use item tray so don't have to have a to make a class

Reply:
was intentionally set as unrandom for testing, since it currently isn't working (he PR'd because he can't test on his computer)
should've noted it in the commenting out of the code though